bin/azure-deploy

```diff
@@ -12,6 +13,8 @@ case $ENVIRONMENT_NAME in
     # Use `az account show --out json` to find the subscription ID.
     SUBSCRIPTION_ID="8655985a-2f87-44d7-a541-0be9a8c2779d"
```

Comment:
I wouldn't normally  have the subscription id in the repo (basically from a security perspective).
Similarly with the resource group name.
Also the name of the file development.json.txt, curious as to why the extension is *.txt

Reply:
Cool, fair enough. We'll ding those two things. On the naming, I think our reasoning was because it's not the actual parameter file, so saves confusion should someone use it in error.

Reply:
perhaps `development.example.json` instead to make it clearer that it's not the real thing?

Reply:
I've renamed it to `development.template.json` since the file is used but with substitutions.

Reply:
I've removed subscription ID and resource group name variables from the script. We'll also need to rotate the values or purge the git history if they're sensitive.

Reply:
Yeah, I think the only way is to purge the history. I'd say those values are pretty much set in stone. That said, I don't think they're highly sensitive, but makes sense to get rid